### DISCUSSION TITLE
Projects (beta): the settings page does not respect browser history

### DISCUSSION THREAD

**Discussion by brettgoulder** [community/community#4734] [Projects and Issues]
**Current behavior**
+ Create a project with two views
+ Navigate from the first view `/views/1` to the second view `/views/2`
+ Click the settings gear button
+ Click the browser's back button

The settings page will remain open but you will see the URL switch from `/views/2` to `/views/1`. 

**Expected behavior**
When I click the back button, I expect to go back to `/views/2` and the settings page to disappear. I believe this can be done simply with an actual route for the settings page, e.g. `/projects/:id/settings`

**Comment by azenMatt** [comment]
Yes! this one is def annoying. We don't have a full fix ready for this yet (with an actual URL) but we are putting in a tempory fix at the moment so that the page starts to respect the back button.

Should be with you next week
